pkg/runctl/error.go

```diff
@@ -0,0 +1,51 @@
+package runctl
+
+import (
+	"errors"
+)
+
+// Error is a runManager error
+type Error struct {
+	e           error
```

Comment:
`e` is a bad field name, as it does not express its meaning.

Reply:
Changed